.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,14 +5,14 @@
 
 ## Explanation
 <!--
-  - Explain what your PR does. If this PR fixes an existing bug, please include 
-  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue 
+  - Explain what your PR does. If this PR fixes an existing bug, please include
+  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
   - when this PR is merged.
   -->
 
 ## Checklist
 - [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes.
-- [ ] The PR explanation includes the words "Fixes #bugnum: ...".
+- [ ] The PR explanation includes the words "Fixes #bugnum: ...". If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".
```

Comment:
The addition here is for the title, so should it go in the previous bullet point?

Also maybe want to put the newly added sentence in parens.

Reply:
done!